d3a_api_client/redis_aggregator.py

```diff
@@ -182,15 +183,18 @@ def execute_batch_commands(self, is_blocking=True):
                 raise RedisAPIException(f'API registration process timed out.')
 
     def _on_event_or_response(self, message):
-        logging.info(f"A new message was received. Message information: {message}")
+        log_msg = copy(message)
+        if 'grid_tree' in log_msg:
+            log_msg['grid_tree'] = '{...}'
```

Comment:
I assume you did that in order to avoid (more than useful) verbose logging, right ?

Reply:
Exactly. I dont think that the grid_tree would be useful on INFO level. I could add another DEBUG log message that contains it?